test/lib/target-frameworks.test.ts

```diff
@@ -54,3 +53,21 @@ test('.Net .fsproj simple project target framework extracted as expected', async
     'manifest.fsproj');
   t.deepEqual(targetFrameworks, ['netcoreapp2.1'], 'targetFramework array is as expected');
 });
+
+/*
+****** packages.config ******
+*/
+
+test('.Net packages.config simple project target framework extracted as expected', async (t) => {
```

Comment:
sorry but can we also have the same test you had before? Same framework in the file, so rather than amend this one add a new one?

Reply:
OK. I've added that additional test now.